internal/logger/logger.go

```diff
@@ -1,3 +1,108 @@
 package logger
 
-// TODO: Implement logger and refactor all the log statements to use it
+import (
+	"fmt"
+	"log"
+	"log/slog"
+	"os"
+)
+
+var LogLevelString = getLogLevel()
+
+// RawLogger is the raw global logger object used for calls wrapped by the logging package
+var RawLogger = InitLogger(LogLevelString, "")
+
+// InitLogger initializes a cluster-id specific child logger
+func InitLogger(logLevelString string, clusterID string) *slog.Logger {
```

Comment:
We want to make this offering generic for non-redhat use - so no need to have special logging for `clusterId`